cli/asc.js

```diff
@@ -718,11 +815,13 @@ exports.main = function main(argv, options, callback) {
   return callback(null);
 
   function readFileNode(filename, baseDir) {
+    let dir = baseDir || path.sep;
```

Comment:
Single `path.sep` seems to be invalid on Windows.

Reply:
It anyway uses as argument for `path.resolve`

Reply:
Then what is?

Reply:
Figured it out.  Now have a `const sep`, which is either the unix version or the windows.